Documentation/dev/developer-guide.md

```diff
@@ -136,6 +136,11 @@ When committing new dependencies, please use the following guidelines:
   - `Gopkg.toml: Add new dependency $your_new_dependency`
   - `Gopkg.lock,vendor: Add new dependency $your_new_dependency`
 
+## Helm templates
+
+If you have added a new Helm Chart and would like to render the template to check the values and nesting within the yaml file you can run:
```

Comment:
Add an empty line after this sentence. This adds a new paragraph/line break in the rendered HTML.